### PR TITLE
[SE-0302] Rename ConcurrentValue/@concurrent to Sendable/@Sendable.

### PR DESCRIPTION
**Pull request by DougGregor**

### CODE REVIEW

**Comment by DougGregor** [comment]
@swift-ci please smoke test and merge

File: include/swift/AST/DiagnosticsSema.def [hunk 1]
```diff
@@ -4404,37 +4404,37 @@ ERROR(global_actor_isolated_requirement_witness_conflict,none,
...
-      "conformance to 'ConcurrentValue' must occur in the same source file as "
-      "%0 %1; use 'UnsafeConcurrentValue' for retroactive conformance",
+      "conformance to 'Sendable' must occur in the same source file as "
+      "%0 %1; use 'UnsafeSendable' for retroactive conformance",
       (DescriptiveDeclKind, DeclName))
 ERROR(concurrent_value_nonfinal_class,none,
-      "non-final class %0 cannot conform to `ConcurrentValue`; "
-      "use `UnsafeConcurrentValue`", (DeclName))
+      "non-final class %0 cannot conform to `Sendable`; "
+      "use `UnsafeSendable`", (DeclName))
```
Comment (ktoso) [inline include/swift/AST/DiagnosticsSema.def:4435]: should these already speak about `@unchecked Sendable` instead?

**Review by ktoso — APPROVED** [review]

File: include/swift/AST/DiagnosticsSema.def (hunk 1 above)
Comment (DougGregor) [reply include/swift/AST/DiagnosticsSema.def:4435]: I didn't get to implementing it yet. When I do, I'll fix these and zap `UnsafeSendable` entirely.

Comment (ktoso) [reply include/swift/AST/DiagnosticsSema.def:4435]: Got it ok, the `UnsafeSendable` type has a deprecation note already hinting at `@unchecked Sendable` so I thought I missed the PR implementing it :-) No problem though, looks good...

**Pull request merged**